src/components/ui/file-upload.tsx

```
@@ -114,16 +117,18 @@ export const FileUpload: React.FC<FileUploadProps> = ({
   return (
     <div>
       <div className="flex items-center justify-between">
-        <label className="text-label text-neutral-foreground">
-          {tipsLabel ?? "Photos"}
+        <label>
+          <Span variant="label">{tipsLabel ?? "Photos"}</Span>
         </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)